app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -33,6 +33,25 @@ message Target {
     // Name of the scheduler to trigger this target.
     // Defaults to being triggered by cocoon.
     optional SchedulerSystem scheduler = 8 [default = cocoon];
+    optional PhaseConfig presubmit = 9;
+    optional PhaseConfig postsubmit = 10;
+}
+
+// CI Phase (i.e. pre-submit vs post-submit).
+// Next ID: 3
+message PhaseConfig {
+    optional Pool pool = 1;
+    // Allows disabling tests not intended to run in this phase.
+    optional bool enabled = 2;
+}
+
+enum Pool {
```

Comment:
We currently have 4 pools for Flutter:
```
luci.flutter.prod
luci.flutter.staging
luci.flutter.try
luci.flutter.tests
```

Should we always mirror from infradata to this proto?

Reply:
hmm, what's tests? and by "mirror", do you mean manually?

Reply:
also, where is that defined? I'll add a comment to the proto linking to it.

Reply:
it looks like luci.flutter.tests is only used for a single fuchsia test? https://cs.opensource.google/search?q=luci.flutter.tests&sq=&ss=flutter

Reply:
@godofredoc do you know what the purpose of `luci.flutter.tests` is?

Reply:
tests is a pool of independent devices that can be used either by try or prod tasks.

Reply:
Thanks! Chris and I synced offline, for now we'll hide pool from `.ci.yaml` and leave that as a detail for the scheduler.